src/js/middleware/index.js

```diff
@@ -0,0 +1,32 @@
+import teamMiddleWare from "./team.js";
+import todoMiddleWare from "./todo.js";
+
+class MiddleWare {
+  constructor() {
+    this.middlewares = {};
+  }
+  fork(type, cb) {
+    return (this.middlewares[type] = cb);
+  }
+
+  has(type) {
+    return this.middlewares[type] ? true : false;
```

Comment:
```suggestion
return !!this.middlewares[type];
```
이렇게만 작성해주셔도 될 것 같습니다 ㅎㅎ !!를 해주는 이유는 값을 boolean으로 처리해주기 위함이에요~

Reply:
와 처음 알아갑니다..!!